_posts/2019-07-10-cockpit-198.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Cockpit 198
+author: pitti
+date: 2019-07-10
+tags: cockpit linux
+slug: cockpit-198
+category: release
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### PatternFly4 user interface design
+
+Cockpit got restyled to match the
```

Comment:
```suggestion
Cockpit has been restyled to match the
```

Using "got" here like this is what a lot of people would say when speaking, but I don't think it's really proper to use when writing. I'm adding a quick change, although I'm not a fan of "has been".

@allisonkarlitskaya: Do you have a better suggestion?